utilities/response.utility.js

```diff
@@ -7,7 +7,7 @@ class JSONResponse {
        });
     }
     
-    static error(res, message = "error", error, status) {
+    static err(res, message = "error", error, status) {
```

Comment:
Why you changed the error method? names should be easily understood which is why it was spelt out.

Reply:
Thats true, i'll take a look at them soon, I also had to fix a few errors, so double check as well ok

Reply:
Understood. However, seeing as though I pulled yours to mine, it made more sense to change that one thing, than to change my entire code to facilitate your's.